phaseshifts/backends.py

```
@@ -29,59 +29,155 @@ def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
         )
 
 
-class ViperLeedBackend(PhaseShiftBackend):
-    """EEASiSSS backend using the ViPErLEED toolchain."""
+def _load_viperleed_modules():
```

Ruff (0.14.10)

Missing return type annotation for private function `_load_viperleed_modules`

(ANN202)